boards/feather-m0/include/board.h

```diff
@@ -54,6 +54,20 @@ extern "C" {
 #define ATWINC15X0_PARAM_WAKE_PIN       GPIO_UNDEF
 /** @} */
 
+/**
+ * @name    Configuration for Feather M0 LoRa and the SX1272 module
```

Comment:
sx1272 or sx1276 ? In the board dependencies it's sx1276 that is used.

Reply:
Adafruit doc says sx1276, so probably a typo. Please fix @tperale